### ISSUE TITLE
AI log cluttered with error message "Pathfinder.cpp:1100 : Passed invalid system id: -1"

### ISSUE THREAD

**Issue by Oberlus** [freeorion/freeorion#1560]
### Environment

* **FreeOrion Version**: 2017-05-02.ded1942_Test_Win32
* **Operating System**: Windows 7
* **Graphic card used**: Nvidia 660 GTX
* **Fetched as** 
  * Binary release
  * Weekly development build

### Description

Some of the AI logs are cluttered with "[error] ai : Pathfinder.cpp:1100 : Passed invalid system id: -1" messages (tens of them for each millisecond). AI_5.log has near 7k lines (I guess 99.9% is that error message) for a session of about 5 turns of playing since FO was started FO and previous game was loaded.
Although I have not experienced any real performance issue, this could be affecting performance on slower machines. Also, it could be the source or the consequence of a bug, and it could be impacting AI's behaviour.

### Expected Result

No occurrence of such errors or at least not repeating it thousands of times per turn.

### Steps to reproduce

No idea, it's on the AI side. I attach logs.
[freeorion_and_ai_logs.zip](https://github.com/freeorion/freeorion/files/996127/freeorion_and_ai_logs.zip)
See issue https://github.com/freeorion/freeorion/issues/1559 for the save game if necessary.

**Comment by geoffthemedio** [comment]
Is this still occurring?

**Comment by Vezzra** [comment]
@Oberlus, bump? As @geoffthemedio asked, is this still occurring?

**Comment by Oberlus** [comment]
Hi,
Sorry, I failed to see Geoff's email during August (whole month away from
home and FO).
Right now I'm in a business trip and I can't check it. I'll try to
replicate the situation and check if it is still occuring. I hope I did
save that game, otherwise I'll just play several games and report back,
next weekend.
Best,
Oberlus.

2017-10-29 15:16 GMT+01:00 Vezzra <notifications@github.com>:

**Comment by Vezzra** [comment]
Ok, thanks!

**Comment by Oberlus** [comment]
I have not been able to reproduce the error.
Latest FO test releases can't load the saves from 12 May.
I've played four games against several AIs to at least turn 250 and well after warfare begins, and there were no occurrences of the Pathfinder.cpp error in any of the AI logs.
Therefore, either I couldn't reproduce the specific situation that triggered this error or the error has been fixed.
I would close this Issue.
